app/server.go

```diff
@@ -827,6 +837,43 @@ func doSessionCleanup(s *Server) {
 	s.Store.Session().Cleanup(model.GetMillis(), SESSIONS_CLEANUP_BATCH_SIZE)
 }
 
+func doStoreAndCheckNumberOfActiveUsersWarnMetricStatus(s *Server) {
+	//check this for TE only
+	license := s.License()
+	if license != nil {
+		mlog.Debug("License is present, skip this check")
+		return
+	}
+
+	warnMetricId := model.SYSTEM_WARN_METRIC_NUMBER_OF_ACTIVE_USERS
+	data, err := s.Store.System().GetByName(warnMetricId)
+	if err == nil && data != nil && (data.Value == "true" || data.Value == "ack") {
+		mlog.Debug("This metric limit crossing has been acknowledged or detected already")
+		return
+	}
+
+	noActiveUsers, err := s.Store.User().Count(model.UserCountOptions{})
```

Comment:
`noActiveUsers` sounds like "no active users" and I think is all the way around. Probably `numberActiveUsers` without any abbreviation here would be better.

Reply:
sure, it it makes it clearer.